dask/dataframe/tests/test_groupby.py

```diff
@@ -26,8 +107,8 @@ def func(df):
 
     assert eq(d.groupby('a').apply(func), full.groupby('a').apply(func))
 
-    assert sorted(d.groupby('a').apply(func).dask) == \
-           sorted(d.groupby('a').apply(func).dask)
+    # assert sorted(d.groupby('a').apply(func).dask) == \
```

Comment:
I guess that that makes sense.  The order to a shuffle is non-deterministic.  It is probably correct that this test fail.